### PR TITLE
Release 0.0.7

### PR DESCRIPTION
**Pull request by AlexYanski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Changelog for CampcotCollectionView 0.0.7
+### Fixed
+* Fixed crash with storyboard realization.
+
+### Added
+* Storyboard support.
+* Properties in `Attributes Inspector` for `CampcotCollectionView`.
+* Example for storyboard realization.
```
Comment (pkandrashkou) [inline CHANGELOG.md]: realization -> implementation

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# Changelog for CampcotCollectionView 0.0.7
+### Fixed
+* Fixed crash with storyboard realization.
```
Comment (pkandrashkou) [inline CHANGELOG.md]: realization -> implementation

File: CHANGELOG.md (hunk 1 above)
Comment (AlexYanski) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (AlexYanski) [reply CHANGELOG.md]: Done

**Review by pkandrashkou — APPROVED** [review]

**Pull request merged**